autoload/lsp/symbol.vim

```diff
@@ -101,7 +101,20 @@ def JumpToWorkspaceSymbol(popupID: number, result: number): void
 	exe $'confirm edit {symTbl[result - 1].file}'
       endif
     else
-      winList[0]->win_gotoid()
+      var wid = fname->bufwinid()
```

Comment:
This can be replaced with the following:
```
     if bufnr() != fname->bufnr()
        var winID = fname->bufwinid()
        if winID == -1
          # not present in the current tab page
          winID = winList[0]
        endif
        winID->win_gotoid()
      endif
    endif
```